forms-native/src/components/RadioButton/index.ts

```diff
@@ -0,0 +1,3 @@
+import RadioButtons from './RadioButtons';
+
+export default RadioButtons;
```

Comment:
hey there! i’m andrew, i came over with the Time Sites acquisiton. i’m just familiarizing myself with Vev and `vev-components` and thought i would look at this PR for an example of how to add a new component set, and everything is surprisingly easy to follow, which is very nice. i noticed here that this is attempting to re-export the default export from `./RadioButtons`, which looks to me like it doesn’t exist (i just see `./RadioButton.tsx`). is that a typo?

Reply:
Hey Andrew! I think this is a feature of the module resolution in TypeScript, meaning it automatically finds the file with either a .ts or .tsx ending. (As far as i understand)

Reply:
@bnhovde thanks for the reply! to clarify: i wasn’t talking about the file extensions, i was talking about the plural (`RadioButtons` vs `RadioButton`).

Reply:
Oh right! Yes, I missed that. That's a typo indeed.